selfservice/strategy/lookup/settings_test.go

```diff
@@ -126,7 +136,14 @@ func TestCompleteSettings(t *testing.T) {
 
 	t.Run("case=hide recovery codes behind reveal button and show disable button", func(t *testing.T) {
 		id, _ := createIdentity(t, reg)
-		browserClient := testhelpers.NewHTTPClientWithIdentitySessionToken(t, reg, id)
+		// TODO: Review this particular diff- I suspect there was a bug since `browserClient` was
+		// actually a token client, not a cookie client
+		browserClient := testhelpers.NewHTTPClientBuilder(t).
+			SetReqestFromWhoAmI().
+			SetIdentity(id).
+			SetSessionDefault().
+			ClientWithSessionCookie(reg)
+			// ClientWithSessionToken(reg)
```

Comment:
I did want to call this change out. Previously, the variable name implied the client should be created with a cookie (for the browser), but was calling the token function.

I just wanted to double check I should be building the client with the cookie and not a token here?

Reply:
I've gone ahead and stuck with using the client w/ cookies on this. Tests seemed to pass 🤷